txscript/opcode.go

```diff
@@ -413,6 +415,8 @@ var opcodeArray = [256]opcode{
 	OP_ENDIF:    {OP_ENDIF, "OP_ENDIF", 1, opcodeEndif},
 	OP_VERIFY:   {OP_VERIFY, "OP_VERIFY", 1, opcodeVerify},
 	OP_RETURN:   {OP_RETURN, "OP_RETURN", 1, opcodeReturn},
+	OP_CHECKLOCKTIMEVERIFY: {OP_CHECKLOCKTIMEVERIFY,
```

Comment:
I know the code is generally kept to 80, but I think for consistency, given every other entry, even those that are longer, are on a single line, this one should be too.